mpqp/core/instruction/measurement/basis_measure.py

```diff
@@ -97,12 +101,12 @@ def __repr__(self) -> str:
             options += (
                 f", basis={self.basis}"
                 if targets and options
-                else f"basis={self.shots}"
+                else f", basis={self.basis}"
```

Comment:
wow, the basis=shots thing before was weird ! but shouldn't we put the shots anyway still ?

Reply:
fixed